_patterns/bjorklund.md

```diff
@@ -1,57 +1,72 @@
 ---
-category: bjorklund 
+category: bjorklund
 layout: default
 ---
 
-If you give two numbers in parenthesis after an element in a pattern, then Tidal will
-distribute the first number of sounds equally across the second number of steps:
+If you give two numbers in parenthesis after an element in a pattern, then Tidal
```

Comment:
Fascinatingly, this website disagrees with me.
 http://www.grammarbook.com/punctuation/parens.asp

I'm pretty sure she's totally wrong though. A parenthesis is the phrase, and you can use brackets to denote one.